### PR TITLE
deleted surv_s_grf.R

### PR DESCRIPTION
**Pull request by CrystalXuR**
Deleted surv_s_grf.R that Nikos wrote for testing grf development version

This resolves to issue #52 remove one S-learner of grf

### CODE REVIEW

File: survlearners/R/surv_s_grf.R [hunk 1]
```diff
@@ -1,20 +0,0 @@
-#' @title S-learner for survival data implemented via grf::survival_forest
-#'
-#' @description  S-learner via grf::survival_forest
-#'
-#' @param data training data
```
Comment (erikcs) [inline survlearners/R/surv_s_grf.R:5]: remember Build->Document

Comment (CrystalXuR) [reply survlearners/R/surv_s_grf.R:5]: ah, good catch!

Comment (CrystalXuR) [reply survlearners/R/surv_s_grf.R:5]: This .Rd file update comes up whenever I delete/add functions/packages, any other situations you have encountered?

Comment (erikcs) [reply survlearners/R/surv_s_grf.R:5]: Yes, any time you touch a `# ' @ ...` section in an R file

**Comment by erikcs** [comment]
@CrystalXuR   you can see what R complains about here:
Screen Shot 2022-04-08 at 13 47 49

**Comment by CrystalXuR** [comment]
yes, I did check it and it says the file 'surv_s_grf' I intended to delete is not available, which I am still trying to understand. I deleted the file manually and wanted to merge that deletion to master. There might be a better way to do this

**Comment by erikcs** [comment]
Do a search in the repo, it is still being called in tests/

**Comment by erikcs** [comment]
Screen Shot 2022-04-08 at 15 09 18

**Comment by CrystalXuR** [comment]
aha, I know where it is!

**Pull request closed without merging**